scheduler/src/helper/generator.helper.ts

```diff
@@ -0,0 +1,423 @@
+import { config as dotenvConfig } from "dotenv";
+import * as fs from 'fs';
+import { join, resolve } from 'path';
+import { CgiSftpService } from "src/modules/cgi-sftp/cgi-sftp.service";
+import { Transaction } from "src/modules/transactions/transaction.entity";
+import { Readable } from "typeorm/platform/PlatformTools";
+import { CgiConstants } from "src/common/constants/cgi.constant";
+
+const envFilePath = resolve(__dirname, "../../../.env");
+const result = dotenvConfig({ path: envFilePath });
```

Comment:
Build failure. Not required. env variables are loaded in app.module.ts.  Remove line 9 and 10